.github/workflows/preview-pr.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+          sudo apt install -y pandoc
```

Comment:
- `nbsphinx `needs `pandocs`. See https://nbsphinx.readthedocs.io/en/0.8.5/installation.html#pandoc
- `pandoc` cannot be installed by `pip`. See https://pandoc.org/installing.html#linux. It could be installed by `conda`.